### ISSUE TITLE
[tracking] Block processing implementations

### ISSUE THREAD

**Issue by MegaRedHand** [lambdaclass/lambda_ethereum_consensus#425]
- [ ] `process_block_header`
- [ ] `process_randao`
- [ ] `process_eth1_data`
- [ ] `process_operations`

**Comment by f3r10** [comment]
can I take this one?

**Comment by MegaRedHand** [comment]
Sure! Feel free to split this into multiple issues if it's too much. Also, keep in mind some of the functions used by these are to be implemented in other issues (for example, most functions inside `process_operations`)

**Comment by Godspower-Eze** [comment]
Isn't `process_eth1_data` already implemented?

**Comment by f3r10** [comment]
@MegaRedHand it is possible to test these functions?

**Comment by MegaRedHand** [comment]
No. The tests for these would be the full state transition suite, but that requires it to be fully implemented.

**Comment by h3lio5** [comment]
can I take the process_operations()?
